### DISCUSSION TITLE
Question about "batch_title_player_telemetry_event" function

### DISCUSSION THREAD

**Discussion by AiurTemplar** [Structed/godot-playfab#143] [Q&A]
Hello guys, I'm still learning this addon and trying `batch_title_player_telemetry_event` vs `write_title_player_playstream_event`, these two functions have the same parameters, but when I invoke `write_title_player_playstream_event` with a callback everything is fine, but invoke `batch_title_player_telemetry_event` with the same callback, callback function will not be triggered.

Then I'm trying to read the code in the addon which is a little bit difficult for me and I think in "PlayFabEvent.gd", line 98 "batch_title_player_telemetry_event", the "callback" parameter is discarded in the function body, is this the problem? Or am I using the addon in the correct way?

Here is the sceenshot of  `batch_title_player_telemetry_event` :
![{@$I(X29Z WL4@D~UM6K6U3](https://github.com/user-attachments/assets/e5a28e57-67bc-4878-89d2-ff7135100f9b)

**Comment by Structed** [comment]
I am so sorry that you had to wait for so long! I never got any notifications on this :(

You should really check out the [Events docs](https://github.com/Structed/godot-playfab/blob/develop/addons/godot-playfab/docs/Events/README.md)! This should answer your questions.


Also, it is true, that the batch methods just disregard the callback. I shall remove the callback, as it's not required, as these methods just batch the vent for later sending - so there is not response.

I have filed an issue on your behalf: #163
